src/Microsoft.AspNetCore.Server.IISIntegration.Tools/WebConfigTransform.cs

```diff
@@ -57,8 +57,10 @@ private static void TransformHandlers(XElement handlersElement)
 
         private static void TransformAspNetCore(XElement aspNetCoreElement, string appName, bool configureForAzure)
         {
-            var appPath = Path.Combine(configureForAzure ? @"%home%\site" : ".", appName);
-            var logPath = Path.Combine(configureForAzure ? @"\\?\%home%\LogFiles" : @".\logs", "stdout.log");
```

Comment:
As per description of the commit: 

_AspNetCoreModule appends a timestamp followed by ".log" to log file name. Removing extension prevents from creating log files with names like stdout.log.{timestamp}.log_